webapp-django/crashstats/crashstats/tests/test_views.py

```diff
@@ -34,37 +34,38 @@
 
 
 _SAMPLE_PROCESSED = {
-    "client_crash_date": "2012-06-11T06:08:45",
-    "signature": "FakeSignature1",
-    "uptime": 14693,
-    "release_channel": "nightly",
-    "uuid": "11cb72f5-eb28-41e1-a8e4-849982120611",
-    "flash_version": "[blank]",
-    "hangid": None,
-    "process_type": None,
-    "id": 383569625,
-    "os_version": "10.6.8 10K549",
-    "version": "5.0a1",
-    "build": "20120609030536",
-    "ReleaseChannel": "nightly",
     "addons_checked": None,
-    "product": "WaterWolf",
-    "os_name": "Mac OS X",
-    "os_pretty_version": "OS X 10.11",
-    "last_crash": 371342,
-    "date_processed": "2012-06-11T06:08:44",
+    "address": "0x8",
+    "build": "20120609030536",
+    "client_crash_date": "2022-06-11T06:08:40",
+    "completed_datetime": "2022-06-11T06:08:50",
     "cpu_arch": "amd64",
     "cpu_info": "AuthenticAMD family 20 model 2 stepping 0 | 2 ",
-    "reason": "EXC_BAD_ACCESS / KERN_INVALID_ADDRESS",
-    "address": "0x8",
-    "completed_datetime": "2012-06-11T06:08:57",
-    "success": True,
-    "user_comments": "this is a comment",
     "crashing_thread": None,
+    "date_processed": "2022-06-11T06:08:44",
+    "flash_version": "[blank]",
+    "hangid": None,
+    "id": 383569625,
     "json_dump": {
         "status": "OK",
         "threads": [],
     },
+    "last_crash": 371342,
+    "os_name": "Mac OS X",
+    "os_pretty_version": "OS X 10.11",
+    "os_version": "10.6.8 10K549",
+    "process_type": "parent",
+    "product": "WaterWolf",
+    "reason": "EXC_BAD_ACCESS / KERN_INVALID_ADDRESS",
+    "release_channel": "nightly",
+    "ReleaseChannel": "nightly",
+    "signature": "FakeSignature1",
+    "success": True,
+    "uptime": 14693,
+    "uuid": "11cb72f5-eb28-41e1-a8e4-849982120611",
+    "version": "5.0a1",
+    # protected data
+    "user_comments": "this is a comment",
```

Comment:
I went through and fixed the test data so it's correct, matches the schema, doesn't have weird things in it, and it's easier to eyeball in regards to public and protected.